src/main/java/ru/practicum/shareit/user/service/UserServiceImpl.java

```diff
@@ -0,0 +1,89 @@
+package ru.practicum.shareit.user.service;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import ru.practicum.shareit.exceptions.CustomValidationException;
+import ru.practicum.shareit.exceptions.ResourceNotFoundException;
+import ru.practicum.shareit.user.dto.UserDto;
+import ru.practicum.shareit.user.mapper.SimpleUserMapper;
+import ru.practicum.shareit.user.model.User;
+import ru.practicum.shareit.user.storage.UserStorage;
+
+import java.util.List;
+import java.util.Optional;
+
+@Slf4j
+@Service
+@RequiredArgsConstructor
+public class UserServiceImpl implements UserService {
+    private final UserStorage userStorage;
+    private final SimpleUserMapper userMapper;
+
+    @Override
+    public User getUserById(int userId) {
+        Optional<User> userOptional = userStorage.getUserById(userId);
+        if (userOptional.isEmpty()) {
+            String msg = "Не нашел пользователя с Id = " + userId;
```

Comment:
При логировании нужно использовать placeholder {}, чтобы подставлять в сообщение параметры. 

```suggestion
            log.error("Не нашел пользователя с Id = {}", userId);
```